src/models.py

```diff
@@ -57,54 +67,17 @@ class Activity(BaseModel, validate_assignment=True):
     time: datetime
     user_id: PositiveInt
     activity_type: ActivityTypes
-    activity_details: str
+    activity_details: str | None
 
     @field_validator("time", mode="before")
     def prevalidate_datetime(cls, entry):
-        if isinstance(entry, int):
-            raise ValueError("Integers are not valid datetime entries")
         if isinstance(entry, str):
             try:
-                print(entry)
+                assert ("Z" in entry) or ("+00:00" in entry)
                 return datetime.fromisoformat(entry)
             except ValueError:
                 raise ValueError(
                     'Incorrect data format, should be "YYYY-MM-DDTHH:MM:SSZ" or "YYYY-MM-DDTHH:MM:SS+00:00" (UTC time)'
```

Comment:
I think the assert statement either passes or returns a `AssertionError` and not a `ValueError`.

In general, just simple if-else (with raise ValueError if condition fails) is good enough here. :)

Reply:
It should be fixed now :)